arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "version": "2017-06-01",
```

Comment:
@manaas-microsoft - So 2017-06-01 is your GA version? If yes, then that's fine. Please note that making changes specially breaking changes are more restrictive after your GA.